src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -60,13 +61,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $serviceId = $input->getArgument('name');
         if ($container->has('workflow.'.$serviceId)) {
             $workflow = $container->get('workflow.'.$serviceId);
+            $dumper = new GraphvizDumper();
         } elseif ($container->has('state_machine.'.$serviceId)) {
             $workflow = $container->get('state_machine.'.$serviceId);
+            $dumper = new StateMachineGraphvizDumper();
```

Comment:
> should we have a way to dump the state machine in workflow mode ? 

For now, it's not really readable to dump a state machine with the workflow dumper. So IMHO, it's not useful to add a way to do it.